### PR TITLE
Fixes breaking post-rebase changes re: ElementType, moving TNS, MNS, …

### PR DESCRIPTION
**Pull request by Gerdie**
Fixes breaking post-rebase changes. Goes with: https://phab.nylas.com/D7433

### CODE REVIEW

File: exchangelib/folders.py [hunk 1]
```diff
@@ -1006,14 +1006,12 @@ def get_default_folder(self, folder_cls):
         # folder was found, try as best we can to return the default folder of type 'folder_cls'
         if not folder_cls.DISTINGUISHED_FOLDER_ID:
             raise ValueError("'folder_cls' %s must have a DISTINGUISHED_FOLDER_ID value" % folder_cls)
-        # Use cached distinguished folder instance, but only if cache has already been prepped. This is an optimization
-        # for accessing e.g. 'account.contacts' without fetching all folders of the account.
-        if self._subfolders:
```
Comment (hallamoore) [inline exchangelib/folders.py:1011]: Why was this removed?

File: exchangelib/services.py [hunk 2]
```diff
@@ -1595,14 +1595,14 @@ def call(self, timeout_minutes):
                                      headers=headers)
 
     def get_payload(self, subscription_id, timeout_minutes):
-        get_streaming_events = create_element('{%s}GetStreamingEvents' % MNS)
-        subscription_ids_elem = create_element('{%s}SubscriptionIds' % MNS)
-        subscription_id_elem = create_element('{%s}SubscriptionId' % TNS)
+        get_streaming_events = create_element('m:GetStreamingEvents')
+        subscription_ids_elem = create_element('m:SubscriptionIds')
+        subscription_id_elem = create_element('t:SubscriptionId')
```
Comment (hallamoore) [inline exchangelib/services.py:1600]: This is technically fine, but why do we want to hardcode these?

File: exchangelib/folders.py (hunk 1 above)
Comment (Gerdie) [reply exchangelib/folders.py:1011]: WELL. This broke test `test_ews_folder_hierarchy_watcher_sync`: previously, we were calling GetFolder on root, then asking for this `root_messages_folder_id` https://github.com/nylas/cloud-core/blob/master/nylas/sync/test/mailsync/backends/ews/integration/test_ews_folder_hierarchy_watcher.py#L45 which would then call `self._folders_map.values` in order to generate `self._folders` on the EWSAccount. When we hide that call behind this if check, we don't generate `self._folders` on the EWSAccount, and end up making individual API calls to get each `WellknownFolder` subfolder of root, instead of generating them en masse here. It's not necessarily a bad if statement, it's just different behaviour that breaks tests and might have implications elsewhere in the codebase

File: exchangelib/services.py (hunk 2 above)
Comment (Gerdie) [reply exchangelib/services.py:1600]: if you're asking why we hardcode the `m` and `t`: That's how the create_element function works now. 🤷‍♀️ 
https://github.com/ecederstrand/exchangelib/commit/304840105aae0b3496ab2c600544cdc0e617a934

if you're asking why we hardcode the create_element calls: That's how we originally implemented this. 🤷‍♀️ https://github.com/nylas/exchangelib/commit/e9a6ed6a671142bc9d5e76a0592f7cccd1898768#diff-ac52c34b5cd482fb648c0fdab773fa99

Comment (hallamoore) [reply exchangelib/services.py:1600]: Oh, I thought `MNS` was the same as `m:` and was very confused about what you meant by "That's how the create_element function works now". I was like yes, but these are equivalent. Turns out they're not! Nevermind~

File: exchangelib/folders.py (hunk 1 above)
Comment (hallamoore) [reply exchangelib/folders.py:1011]: Ah, I see. I don't think it would have weird implications in our code base, but I do think it's weird that `get_folder()` just directly invokes `_folder_map` while `get_default_folder()` only invokes it if it's already been invoked and cached. I think this makes more sense :+1:

**Review by hallamoore — APPROVED** [review]
